### DISCUSSION TITLE
MOD3.0 Design issue #1: What class of DCAT does a Semantic Artefact specialize?

### DISCUSSION THREAD

**Discussion by jonquet** [sifrproject/MOD-Ontology#34] [General]
mod:SemanticArtefact subclass of both classes

- a resource (« something that can be catalogued » 
- a dataset (« collection of data ») … in the sense a collection of classes, concepts, instances, properties as captured to represent the knowledge

<img width="269" alt="image" src="https://user-images.githubusercontent.com/7379183/231006335-c16de34a-f19c-4e26-8a80-1a1c3401ccc2.png">

**Comment by jonquet** [comment]
Properties coming from DCAT
<img width="199" alt="image" src="https://user-images.githubusercontent.com/7379183/231006688-71a9ea8f-356b-4496-a8c9-91e9b1485eac.png">

**Comment by agbeltran** [comment]
In terms of defining ```mod:SemanticArtefact``` as a subclass of ```dcat:Resource``` and ```dcat:Dataset```, I would point out that:
- it certainly makes sense that a ```mod:SemanticArtefact```is a```dcat:Dataset``` so that we can make use of the ```dcat:distribution``` property to point to the specific representations of the semantic artefact (```dcat:Distribution```)
- while we can inherit from both classes, I would keep it simple and only inherit from ```dcat:Dataset``` and not ```dcat:Resource```, as that relationship will be inferred anyway

So, I suggest as a decision for this design issue that we add 

```
mod:SemanticArtefact
  a rdfs:Class ;
  a owl:Class ;
  rdfs:subClassOf dcat:Dataset ;
```

**Reply by jonquet** [reply]
I mostly agree with the formal aspect (it would be infer). 
Originally I mostly used the diagrams to illustrate the idea. And it was important that one can see the fact that a SemanticArtefact (also called a Semantic Resource) is a Resource.

**Comment by antony-wilson** [comment]
I also like the idea of keeping it simple and just inheriting from `dcat:Dataset`

**Comment by agbeltran** [comment]
Just a note to link to the discussion on the design issue 2, which refers to the point of distributions above https://github.com/FAIR-IMPACT/MOD/discussions/35

**Comment by agbeltran** [comment]
To contribute to this discussion, the definition of ```dcat:Dataset``` is "A collection of data, published or curated by a single agent, and available for access or download in one or more representations." (see https://w3c.github.io/dxwg/dcat/#Class:Dataset)

**Reply by jonquet** [reply]
FAIRsFAIR discussed about 3 years ago the fact of considering a Semantic Artefact a Dataset and the decision was to do it in order to be able to rely on DCAT for the next iteration of MOD. 
The discussion also happened later in FAIR-IMPACT but without another concrete solution.

**Comment by agbeltran** [comment]
Further to the discussion on the FAIR-IMPACT meeting this week (that I aim to summarise here when I get a chance), there is further discussion on this on the DCAT issue tracker: https://github.com/w3c/dxwg/issues/1576

**Comment by mariapoveda** [comment]
I wouldn't agree on having all types of semantic artefacts as datasets while some are (like skos vocabularies) for the case of ontologies doesn't seem natural to be classified as datasets. I would remove the subclassOf between SemanticArtefact and Dataset

**Comment by agbeltran** [comment]
Summary of the discussion and approach:

We had multiple discussions on this design issue (FAIR-impact project meetings related to T4.2.3), the discussion above, as well as discussion in the DCAT issue https://github.com/w3c/dxwg/issues/1576).

While some team members are happy with deriving ```mod:SemanticArtefact``` from ```dcat:Dataset```, others prefer to distinguish between a dataset and a semantic artefact, so prefer to derive from ```dcat:Resource``` that is DCAT's extension point.

However, for semantic artefacts, we still want to take advantage of the dichotomy between abstract entity and distribution, so the suggestion is that we will still use the ```dcat:distribution```property for semantic artefacts. In effect, this implies that ```dcat:SemanticArtefact```can be inferred as a ```dcat:Dataset```in the current interpretation of DCAT. This seemed to be a good compromise with those who disagree to make the subclass explicit.

Thus, we will represent:
```
mod:SemanticArtefact
  a rdfs:Class ;
  a owl:Class ;
  rdfs:subClassOf dcat:Resource ;
```
and still use the distribution predicate to refer to multiple representations of the semantic artefact.

**Reply by agbeltran** [reply]
I updated the issue https://github.com/FAIR-IMPACT/MOD/issues/44

I also note the point raised by @jonquet about the schema.org term for ```sdo:DefinedTermSet```, see https://schema.org/DefinedTermSet, which again distinguishes betwee Dataset and a set of defined terms

https://github.com/FAIR-IMPACT/MOD/discussions/51

**Comment by agbeltran** [comment]
Closing this discussion as the conclusion is available as an issue: https://github.com/FAIR-IMPACT/MOD/issues/44
